### PR TITLE
add target arg for clang

### PR DESCRIPTION
**Pull request by conorpp**

### CODE REVIEW

**Comment by nickray** [comment]
When is this needed? Does this work when no TARGET is set in env?

**Comment by conorpp** [comment]
It doesn't work for MacOS unless the target is set

File: build.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@ use std::path::PathBuf;
 
 fn main() {
     let mut builder = cc::Build::new();
+    let target = env::var("TARGET").unwrap_or("".to_string());
```
Comment (nickray) [inline build.rs]: I think `env::var("TARGET")?` as in micro-ecc-sys should work here (TARGET is always set).

**Review by nickray — CHANGES_REQUESTED** [review]

**Comment by nickray** [comment]
Do you need a release here (and in micro-ecc-sys)?

**Comment by conorpp** [comment]
Yeah that would be nice

**Pull request merged**